packages/web3-providers-http/src/index.js

```diff
@@ -121,7 +131,7 @@ HttpProvider.prototype.send = function (payload, callback) {
         request.send(JSON.stringify(payload));
     } catch(error) {
         this.connected = false;
-        callback(errors.InvalidConnection(this.host));
+        callback(errors.InvalidConnection(this.host, { code: 'ECONNREFUSED', reason: 'ECONNREFUSED' }));
```

Comment:
Is `code` supposed to be an HTTP response code?

Reply:
sử dụng N and P

Reply:
@spacesailor24 no, this is a connection error, so the response code is zero.